### PR TITLE
Jinja2: update to version 2.11.2

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: @dangowrt 
Compile tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master
Run tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master

Description:
- Update to version [2.11.2](https://jinja.palletsprojects.com/en/2.11.x/changelog/#version-2-11-2)
- Update project website

### CODE REVIEW

File: lang/python/Jinja2/Makefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ define Package/python3-jinja2
   CATEGORY:=Languages
   SUBMENU:=Python
   TITLE:=Jinja2
-  URL:=http://jinja.pocoo.org/
+  URL:=https://jinja.palletsprojects.com/en/2.11.x/
```
Comment (karlp) [inline lang/python/Jinja2/Makefile:28]: I'd say https://jinja.palletsprojects.com/ is a better option here, even if it's currently redirecting to the versioned sub tree.  or even https://palletsprojects.com/p/jinja/ which links "jinja.palletsproject.com" as the url for docs

Comment (BKPepe) [reply lang/python/Jinja2/Makefile:28]: Thanks @karlp for review. I was not quite sure about it here.
Like you are saying there is an versioned subtree, but there could be released version 2.12 and on the router, there is installed 2.11 and it woudn't match. 

The website I was thinking for that and skipped that. Don't know the reason now, but maybe that the previous URL pointed to doc.

Anyway, I don't mind putting there https://palletsprojects.com/p/jinja/ .
Is that OK for you?

Comment (karlp) [reply lang/python/Jinja2/Makefile:28]: works for me, I just meant that in general we link to the _project_ page, not a particular version.

Comment (BKPepe) [reply lang/python/Jinja2/Makefile]: Changed. If you agree, please resolve the discussion and I will merge it.

**Comment by dangowrt** [comment]
@BKPepe Thanks for taking care of that. I have stopped using those Python packages myself (Jinja2, Flask, Werkzeug) and would appreciate transferring maintainership to you or eg. @aparcar (or anyone else actually using them). Interested?

**Comment by BKPepe** [comment]
I think for Turris routers I'm interested in Flask and Werkzeug as we are using it for our redesigned interface, which is the development. Jinja2 is still used for the previous one, so for some time I can send updates, but as being maintainer don't know for this one.

File: lang/python/Jinja2/Makefile (hunk 1 above)
Comment (karlp) [reply lang/python/Jinja2/Makefile]: oh, sorry, I had only meant to comment, not "start a review" :)

**Pull request merged**